### PR TITLE
stuff

### PR DESCRIPTION
**Pull request by FloppyDolphin57**
onic86 hac

### CODE REVIEW

**Comment by Intoprelised** [comment]
There is a way you can use enchant how it is but this will help

File: Horion/DrawUtils.cpp [hunk 1]
```diff
@@ -372,13 +372,16 @@ void DrawUtils::drawNameTags(C_Entity* ent, float textSize, bool drawHealth, boo
 		rectPos.w = textPos.y + textHeight + 2.f * textSize;
 		vec4_t subRectPos = rectPos;
 		subRectPos.y = subRectPos.w - 1.f * textSize;
-		fillRectangle(rectPos, MC_Color(13, 29, 48), 0.8f);
-		fillRectangle(subRectPos, MC_Color(28, 107, 201), 0.9f);
+		auto nametagsMod = moduleMgr->getModule<NameTags>();
```
Comment (GodGamer029) [inline Horion/DrawUtils.cpp]: should be static

File: Horion/Module/Modules/Xray.cpp [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+		if (!gotSmoothInfo) {
+			gotSmoothInfo = true;
+			wasSmooth = *smoothLightningSetting;
+		}
+		*smoothLightningSetting = 0;
+	}
+}
+
+void Xray::onTick(C_GameMode* gm) {
+	onEnable();  // yes lol
```
Comment (GodGamer029) [inline Horion/Module/Modules/Xray.cpp]: dude what

File: Memory/Hooks.cpp [hunk 3]
```diff
@@ -38,17 +38,16 @@ void Hooks::Init() {
...
-			int offset = *reinterpret_cast<int*>(sigOffset + 3);
-			uintptr_t** blockLegacyVtable = reinterpret_cast<uintptr_t**>(sigOffset + offset + 7);
-			if (blockLegacyVtable == 0x0 || sigOffset == 0x0)
-				logF("C_BlockLegacy signature not working!!!");
-			else {
-				g_Hooks.BlockLegacy_getRenderLayerHook = std::make_unique<FuncHook>(blockLegacyVtable[180], Hooks::BlockLegacy_getRenderLayer);
-				g_Hooks.BlockLegacy_getLightEmissionHook = std::make_unique<FuncHook>(blockLegacyVtable[183], Hooks::BlockLegacy_getLightEmission);
-			}
-		}
+        {
```
Comment (GodGamer029) [inline Memory/Hooks.cpp]: while you're at it, fix this formatting

File: Memory/Hooks.cpp [hunk 4]
```diff
@@ -239,8 +238,8 @@ void Hooks::Init() {
 		void* chestScreenControllerTick = reinterpret_cast<void*>(FindSignature("48 89 5C 24 08 57 48 83 EC 20 48 8B F9 E8 ?? ?? ?? ?? 48 8B 17 48 8B CF 8B D8 FF 92 ?? ?? ?? ?? 84 C0 74 31"));
 		g_Hooks.ChestScreenController_tickHook = std::make_unique<FuncHook>(chestScreenControllerTick, Hooks::ChestScreenController_tick);
 
-		void* fullbright = reinterpret_cast<void*>(FindSignature("48 83 EC ? 80 B9 ? ? ? ? ? 48 8D 54 24 ? 74 ? 41 B8 ? ? ? ? E8 ? ? ? ? 48 8B 10 48 85 D2 74 ? 48 8B 42 ? 48 8B 88 ? ? ? ? 48 85 C9 74 ? E8 ? ? ? ? 48 83 C4 ? C3 F3 0F 10 42"));
-		g_Hooks.GetGammaHook = std::make_unique<FuncHook>(fullbright, Hooks::GetGamma);
+		void* fullbright = reinterpret_cast<void*>(FindSignature("48 83 EC ?? 80 B9 ?? ?? ?? ?? ?? 48 8D 54 24 ?? 48 8B 01 74 35 41 B8 0D 01 00 00"));
+        g_Hooks.GetGammaHook = std::make_unique<FuncHook>(fullbright, Hooks::GetGamma);
```
Comment (GodGamer029) [inline Memory/Hooks.cpp]: formatting

**Review by GodGamer029 — CHANGES_REQUESTED** [review]
Good changes, just a few formatting problems here and there

**Comment by Thekingpenguins** [comment]
close da pr is sexy pr

**Comment by Synx173** [comment]
epic

**Pull request merged**